packages/main/src/ToggleButton.js

```diff
@@ -67,6 +68,12 @@ class ToggleButton extends Button {
 	_onclick() {
 		this.pressed = !this.pressed;
 	}
+
+	_onkeyup(event) {
```

Comment:
And what about the _onkeyup in Button.js:

```js
_onkeyup(event) {
		if (isSpace(event) || isEnter(event)) {
			this.active = false;
		}
	}
```
	
that is inherited by the ToggleButton, but now overwritten by:

```js
_onkeyup(event) {
		if (isSpaceShift(event)) {
			event.preventDefault();
		}
	}
```

I guess the ToggleButton still needs the SPACE, ENTER handling onkeyup, but now it will be lost with this change as you introduce _onkeyup in the ToggleButton, that will overwrite the _onkeyup in the Button. Perhaps you need a mix between the two or I don't understand the change.

Reply:
Done